packages/flutter/lib/src/widgets/overlay.dart

```diff
@@ -5,14 +5,43 @@
 import 'basic.dart';
 import 'framework.dart';
 
-/// A place in the overlay that can contain a widget.
+/// A place in an [Overlay] that can contain a widget.
```

Comment:
I think "place" is better here because it suggests a physical location.  "Entry" would just duplicate the class name :)